celery/backends/filesystem.py

```diff
@@ -61,10 +61,10 @@ def __init__(self, url=None, open=open, unlink=os.unlink, sep=os.sep,
     def _find_path(self, url):
         if not url:
             raise ImproperlyConfigured(E_NO_PATH_SET)
-        if url.startswith('file:///'):
-            return url[7:]
         if url.startswith('file://localhost/'):
             return url[16:]
+        if url.startswith('file://'):
```

Comment:
Im seeing just a ordering of if statements change, do we have any tests to cover this @watercrossing ?

Reply:
I have not had time to look into celery's test setup. The re-ordering of if statements is necessary to preserve current behaviour; otherwise any `url` starting with `'file://localhost/' would not get the `'localhost'` striped.

Reply:
please also add tests for the change :)